src/app/article/article/article.component.ts

```diff
@@ -11,18 +12,39 @@ import { ArticleService } from 'src/app/article/article.service';
 })
 export class ArticleComponent implements OnInit {
   article$ = this.articleService.article$;
+  comments$ = this.articleService.comments$;
+  user$ = this.articleService.user$;
   favoritedArticle$ = this.article$.pipe(map((article) => article?.favorited));
 
+  isCreatedByCurrentUser$ = this.user$.pipe(
+    concatMap((user) =>
```

Comment:
Just for information: there are concatMap and mergeMap https://www.learnrxjs.io/learn-rxjs/operators/transformation/concatmap but I haven't checked which one is better in your case.

Reply:
Thanks